ADR/Components/Propulsion/test/test_Motor.py

```diff
@@ -0,0 +1,7 @@
+from ADR.Components.Propulsion.Motor import *
+import numpy.testing as npt
+
+
+def test_Motor():
+    mydict = {"static_thrust": 14, "linear_decay_coefficient": 2}
+    npt.assert_almost_equal(Motor(mydict).thrust(35), -56, decimal=0)
```

Comment:
Nice! It indeed works!

Let's just make some changes to make the code more readable:
- change the variable "mydict" to some more meaningful, like "motor_parameters"
- instantiate the motor on it's own line, like motor = Motor(motor_parameters), so it gets clearer
- test the thrust in some velocity that gives positive thrust. It doesn't really matters, but negative thrust seems strange for a motor hehe